### PR TITLE
Bump versions to 0.38.0

### PR DESCRIPTION
**Pull request by digorithm**

### CODE REVIEW

**Review by Braqzen — DISMISSED** [review]

**Comment by iqdecay** [comment]
We haven't updated the "Setting up" mdbook page since 0.32, can you please update it? Look for the `0.32` string

**Review by iqdecay — CHANGES_REQUESTED** [review]
Please update `setup.md` file

**Comment by digorithm** [comment]
Ugh, my search-and-replace is failing me hah.

File: docs/src/getting-started/setup.md [hunk 1]
```diff
@@ -67,10 +67,10 @@ cargo test -- --nocapture
 Add these dependencies on your `Cargo.toml`:
 
 ```toml
-fuels = "0.32"
+fuels = "0.38"
 ```
 
-> **Note** We're using version `0.32` of the SDK, which is the latest version at the time of this writing.
+> **Note** We're using version `0.38` of the SDK, which is the latest version at the time of this writing.
```
Comment (iqdecay) [inline docs/src/getting-started/setup.md:73]: > **Note** We're using version `0.38` of the SDK, which is the latest version at the time of this writing.

**Review by iqdecay — CHANGES_REQUESTED** [review]

**Review by iqdecay — APPROVED** [review]
LGTM, releaaaaaaase the kraken

**Pull request merged**